src/Frame.js

```diff
@@ -0,0 +1,96 @@
+function Frame() {
+  this.pins = 10
+  this.currentFrame = []
+  this.frameScores = []
+  this.turnCount = 0
+  this.spare = false
+  this.strike = false
+  this.game_over = false
+  this.points = 0
+}
+
+Frame.prototype.bowl = function() {
```

Comment:
This function has too many statements. (21)<br>This function's cyclomatic complexity is too high. (8)